### ISSUE TITLE
Missing titles in RoyalRoadL.com & problems with imgur.com images

### ISSUE THREAD

**Issue by buthub** [dteviot/WebToEpub#121]
Hi, fantastic work on this incredible addon!

Recently there seems to be a problem with the titles in royalroadl.com.  It now only does <unknown> for all the ones I've tried.  It happens on both Chrome & Firefox.

Also, I'm not sure if it's a problem with calibre or WTE, but when converting from epub using calibre, some chapters get a proper <p></p> tags for their paragraphs, while other chapters don't.  I'm still not sure why it seems to happen almost randomly within the same ebook... 

As well, it seems to have trouble getting images if they're hosted by imgur.com.  It always says that it found an html file when expecting an image, unless there's something about the options that I got wrong?  Also happens on both FF & Chrome (latest).

Thanks.  This addon is awesome!  Keep up the great work!

**Comment by dteviot** [comment]
@buthub
I'm aware of the imgur problem.  Format changed, so parser was failing to recognise it.  I've got a fix in the Experimental Tab branch, will be merging to trunk shortly.  For time being you could go to https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode and follow the "How to install from Source" instructions.  Note, you'll need to download the source from the ExperimentalTabMode branch.

> Recently there seems to be a problem with the titles in royalroadl.com. It now only does for all the ones I've tried. It happens on both Chrome & Firefox.

Can you be more specific?  What's the problem?  Give me a couple of URLs, and describe (1) what happens and (2) what you think should happen.

> some chapters get a proper tags for their paragraphs, while other chapters don't.

Sorry, your text got mangled.  Probably because you typed &lt; or &gt;  you need to use &amp;lt;  &amp;gt; instead.
Again, if you can provide URLs,  working and not working plus expected and actual behaviour, that will save me time.

Thanks.

**Comment by dteviot** [comment]
@buthub
> Recently there seems to be a problem with the titles in royalroadl.com. It now only does for all the ones I've tried. It happens on both Chrome & Firefox.

Ah.  Now I see, it's not finding the story title, which is due to it changing from a &lt;h2&gt; to a &lt;h1&gt;

> some chapters get a proper tags for their paragraphs, while other chapters don't.

You'll need to give me more specific examples, with URLs so I can reproduce the problem.  I don't have time to go searching.

**Comment by dteviot** [comment]
@buthub
Have updated version in Chrome store to 0.0.0.37.  
Your installed version should update within 24 hours.
This has fixes for the imgur and RoyalRoadL Titles problems you've raised.
Does not have fix for the paragraph issue.
Will load new Firefox version to Mozilla site in a few days.

**Comment by buthub** [comment]
Heh, thanks.  Noticed it stopped, but didn't realize it was because you'd fixed it.  Thanks.  

I just tested it in Firefox and it still has the title problem.  Guess the fix will come later for FF.
The pages I tested on was,
http://royalroadl.com/fiction/10264/soulstone-the-skeleton-king
http://royalroadl.com/fiction/6133/wizard-with-the-flower-blades
http://royalroadl.com/fiction/6957/no-respawn
Chrome was working fine for all three.
